Only remove the leading forward slash from Docker container names

TheConnMan opened the pull request:
Linked container names can be of the form /parent-name/container-name. The previous implementation would only split once starting from the end to pull the child container name only (container-name). This change will only remove the leading / to get parent-name/container-name.

Closes #62

Grumblesmith commented:
@TheConnMan what happens if the container has more than one name in the names array?

TheConnMan commented:
Which names array? I don't understand your question.

Grumblesmith commented:
The container names array from the Docker API

TheConnMan commented:
I'm not modifying any names array. The name in this code change is a single string. Are you asking about this name array? https://github.com/Netuitive/netuitive-diamond/blob/8ee12af73eb183fbc452a6641937dc4bcadeffce/src/collectors/netuitivedocker/netuitivedocker.py#L131

TheConnMan committed:
Use the Docker container name without linked parent container names

All linked container names contain the parent and child container name
(the child container name as it's seen by the linked parent, which is
not necessarily the same as the child container's name as seen by
Docker). The linked containers have two forward slashes, so the child
container name as seen by Docker is the only one with a single forward
slash.

Closes #62
